Shipfinity.Api/appsettings.json

```diff
@@ -7,6 +7,6 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "ShipfinityDbString": "Data Source=localhost\\SQLEXPRESS;Database=ShipfinityDb;Integrated Security=True;Connect Timeout=30;Encrypt=False;Trust Server Certificate=False;Application Intent=ReadWrite;Multi Subnet Failover=False"
+    "ShipfinityDbString": "Data Source=(localdb)\\LocalDBDemo;Database=ShipfinityDbTest;Integrated Security=True;Connect Timeout=30;Encrypt=False;Trust Server Certificate=False;Application Intent=ReadWrite;Multi Subnet Failover=False"
```

Comment:
Do not push changes to appsettings.json.
Keep changes locally unless adding not variables